pkgs/tools/nix/alejandra/default.nix

```
@@ -27,6 +27,6 @@ rustPlatform.buildRustPackage rec {
     homepage = "https://github.com/kamadorueda/alejandra";
     changelog = "https://github.com/kamadorueda/alejandra/blob/${version}/CHANGELOG.md";
     license = licenses.unlicense;
-    maintainers = with maintainers; [ _0x4A6F kamadorueda ];
+    maintainers = with maintainers; [ _0x4A6F kamadorueda Sciencentistguy ];
```

I think this broke evaluation.
```suggestion
    maintainers = with maintainers; [ _0x4A6F kamadorueda sciencentistguy ];
```